### PR TITLE
[PET-17] Implement .toggle--menu

### PR DESCRIPTION
**Pull request by joshuawaheed**

### CODE REVIEW

File: src/stylesheets/components/_list-styles.scss [hunk 1]
```diff
@@ -301,3 +301,27 @@ footer .list {
...
+    @include milli;
+    margin-right: -($base-spacing-unit-small);
+    margin-left: -($base-spacing-unit-small);
+    @include media($small) {
+      column-count: 2;
+      column-gap: 0;
+    }
+  }
+  li {
+    // Stops LIs from leaking into the next column
```
Comment (naseberry) [inline src/stylesheets/components/_list-styles.scss]: You can remove vendor prefixes as the npm module 'autoprefixer' will do this.

File: src/stylesheets/components/_buttons.scss [hunk 2]
```diff
@@ -110,3 +110,30 @@
...
+  }
+  .btn--close {
+    float: right;
+    @include links($grey-1, none, $grey-2, $grey-1);
+    @include milli;
+    margin-bottom: 0;
+  }
+}
+
+body:not(.has-js) .toggle--menu {
```
Comment (naseberry) [inline src/stylesheets/components/_buttons.scss]: I think we need rid ourselves of the `:not()` pseudo class by having classes for each state (`.has-js` or `.no-js`) that we can target

File: src/stylesheets/components/_toggles.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }
+}
+
+body:not(.has-js) .toggle--menu > .block {
+  display: block;
+}
+
+/*--- Functional ---*/
+
+[data-toggle="content"]:not(.open) {
```
Comment (naseberry) [inline src/stylesheets/components/_toggles.scss]: I think we need rid ourselves of the `:not()` pseudo class by having classes for each state (`.open` or `.close`) that we can target

File: src/stylesheets/components/_buttons.scss (hunk 2 above)
Comment (joshuawaheed) [reply src/stylesheets/components/_buttons.scss]: Will skip this for now as it's a larger project and will be better to treat as a separate project.

File: src/stylesheets/components/_toggles.scss (hunk 3 above)
Comment (joshuawaheed) [reply src/stylesheets/components/_toggles.scss]: Will also skip this for now as it's a larger project and will be better to treat as a separate project.

**Pull request merged**